.docker/experimental/Dockerfile

```diff
@@ -0,0 +1,29 @@
+# moveit/moveit:kinetic-experimental
+# Based on a moveit source install, adds the mongo driver and the warehouse packages
+
+FROM moveit/moveit:kinetic-source
```

Comment:
if you build on top of kinetic-source you should use CATKIN_WS = /root/ws_moveit like the underlying container did